models/src/main/java/com/vimeo/networking2/enums/TrialType.kt

```diff
@@ -0,0 +1,17 @@
+package com.vimeo.networking2.enums
+
+/**
+ * Status types for a user's trial.
+ */
+enum class TrialType(override val value: String?) : StringValue {
+
+    /**
+     * Unknown trial type.
+     */
+    UNKNOWN(null),
```

Comment:
This should be the list last in the enum to consistency follow the setup in other DTOs.